arelle/PluginManager.py

```diff
@@ -71,6 +72,11 @@ def init(cntlr: Cntlr, loadPluginConfig: bool = True) -> None:
         pluginConfigChanged = False # don't save until something is added to pluginConfig
     modulePluginInfos = {}  # dict of loaded module pluginInfo objects by module names
     pluginMethodsForClasses = {} # dict by class of list of ordered callable function objects
+    unloaded_arelle_plugins = entry_points(group='arelle.plugin')
+    if unloaded_arelle_plugins:
+        for unloaded_plugin in unloaded_arelle_plugins:
+            plugin_url = unloaded_plugin.load()
+            addPluginModule(plugin_url())
```

Comment:
While the plugin architecture should permit loading pip installed plugins, I think it should still require enabling the plugin via the command line or GUI

Reply:
Discussed a bit offline. I think we landed on passing the plugin name to enable it still